From 9375d4f0f02b4e633248ad340ad111170f856493 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 30 Jan 2025 11:14:03 -0800
Subject: [PATCH] Assume gnulib location

---
 bootstrap | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/bootstrap b/bootstrap
index a84eb39..54a6e67 100755
--- a/bootstrap
+++ b/bootstrap
@@ -37,9 +37,6 @@ done
 
 case ${GNULIB_SRCDIR--} in
 -)
-  echo "$0: getting gnulib files..."
-  git submodule init || exit $?
-  git submodule update || exit $?
   GNULIB_SRCDIR=.gnulib
   ;;
 *)
-- 
2.47.1

